lens/sqlrepo/repo.go

```diff
@@ -21,6 +22,11 @@ func NewAPIOpener(c *cli.Context) (lens.APIOpener, lens.APICloser, error) {
 		LogCacheStatsOnUSR1:     true,
 	}
 
+	enablePreload := os.Getenv("LOTUS_CHAINSTORE_PRELOAD_RECENTS")
```

Comment:
Can you use `c.String("lens-postgres-preload")` so it's consistent with other usages here and then add it to the cli command. Trying to avoid lots of unduscoverable env variables - adding them to the cli command means the user can see them with `--help`